lib/shopify_app/controller_concerns/ensure_billing.rb

```diff
@@ -55,6 +56,7 @@ def has_active_payment?(session)
     end
 
     def has_subscription?(session)
+      ShopifyApp::Logger.debug("Checking if has subscription")
```

Comment:
checking if what has subscription?

Reply:
Yeah as far as I can tell thats whats happening, "Checking if shop has subscription" would probably be more clear.